omnigan/fire.py

```diff
@@ -0,0 +1,81 @@
+import torch
+import numpy as np
+import torch.nn.functional as F
+from PIL import Image, ImageEnhance, ImageFilter
+from torchvision import transforms
+
+
+def normalize(arr, mini=0, maxi=1):
+    return mini + (maxi - mini) * (arr - arr.min()) / (arr.max() - arr.min())
+
+
+def retrieve_sky_mask(seg_preds):
+    seg_ind = torch.argmax(seg_preds.squeeze(), dim=0)
```

Comment:
check that seg_preds has 4 dimensions, if ever we want to use greound truth this will break, as it will only have 3 dims with ints and so don't do the argmax

Reply:
Good point,done!